### PR TITLE
Properly highlight subscription cell

### PR DESCRIPTION
**Pull request by legendecas**
@RocketChat/iOS

### CODE REVIEW

**Comment by legendecas** [comment]
Ok, I'll take a look.

File: Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift [hunk 1]
```diff
@@ -330,12 +330,27 @@ extension SubscriptionsViewController: UITableViewDelegate {
...
+
     func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         let subscription = self.groupSubscriptions?[indexPath.section][indexPath.row]
         let controller = ChatViewController.sharedInstance()
         controller?.closeSidebarAfterSubscriptionUpdate = true
         controller?.subscription = subscription
     }
+
+    func tableView(_ tableView: UITableView, willDisplay cell: UITableViewCell, forRowAt indexPath: IndexPath) {
+        guard let sell = cell as? SubscriptionCell else { return }
```
Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: `sell`? 😂 You can use `cell` here.

Comment (legendecas) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: `cell` is `UITableViewCell` but not `SubscriptionCell` so it did not have property `subscription`.

Comment (legendecas) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: @rafaelks I don't think I can write typecasted value to immutable variable `cell`.

Comment (rafaelks) [inline Rocket.Chat/Controllers/Subscriptions/SubscriptionsViewController.swift]: Gotcha, can you change to something like `subscriptionCell` then?

**Pull request merged**